recipes/antismash/post-link.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 # download the databases required by several options
-$PREFIX/bin/download-antismash-databases
```

Comment:
Did it fail with the `$PREFIX/bin/` in place? IIRC, older `conda` versions (`<4.6`) did not activate the environment. But `PREFIX` should be available in the script, no matter which `conda` version.

Reply:
It seems to have failed then, yes.

Reply:
The link scripts are very awkward in general... Since the previous version with `PREFIX` would not have worked reliably for `conda <4.6` users, I'm okay with this change. (If the users run the install in an already activated environment, everything should be fine.)